### DISCUSSION TITLE
pi-Base community call Thu 2024 July 11 at 3pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#636] [Announcements]
Agenda:

- potential `notation:` and `keywords:` fields https://github.com/pi-base/web/issues/159
- "Has a" prefix in certain property names
    - In meeting, we determined there wasn't a great rule for when this is appropriate or not, so no action will be taken
- consistent capitalization of property names: "Second Countable" --> "Second countable", etc
    - @prabau volunteered to handle this chore, using the rule that only first words and proper nouns should be capitalized.
- how to choose among multiple sufficient theorem options: https://github.com/pi-base/data/pull/679#issuecomment-2201834851
    - We started an experiment here, but more work is needed so theorems don't accidentally refer to themselves: https://github.com/pi-base/data/pull/689
- threshold for modifying results from literature without external discussion (e.g. without posting to Math SE): https://github.com/pi-base/data/pull/676
- discuss https://github.com/pi-base/data/pull/658
- teach new contributors how to contribute instead of doing steps for them'
    - We recorded a video: https://youtu.be/-us3rzXjGFk

Zoom link: https://southalabama.zoom.us/j/93833246214

**Comment by StevenClontz** [comment]
Hi @prabau and @ccaruvana - sorry I was unexpectedly off-grid yesterday afternoon.

What do we think about Thu Jun 27 for the next meeting?

**Reply by prabau** [reply]
> What do we think about Thu Jun 27 for the next meeting?

Good for me

**Reply by ccaruvana** [reply]
I'll be generally away 26 June to 1 July. I will also have to double check on the dates, but I'll be generally away for a week and half in early July, as well.

**Reply by prabau** [reply]
Jun 20 seems a little too early if there are no pending issues.  How about marking it tentatively to some date in mid July?

**Reply by StevenClontz** [reply]
Thu July 11? (I'm unavailable the Thursday before and after.)

**Reply by prabau** [reply]
Jul 11 works for me.

**Reply by StevenClontz** [reply]
I've edited the title of this thread for the new date of July 11.

**Comment by prabau** [comment]
@StevenClontz Contributor @Almanzoris has great intentions, but he seems quite lost in how to do things efficiently with github.  What do you think about inviting him to the zoom meeting and giving him a demo on how to use github.dev, etc?

**Reply by prabau** [reply]
> @StevenClontz Contributor @Almanzoris has great intentions, but he seems quite lost in how to do things efficiently with github. What do you think about inviting him to the zoom meeting and giving him a demo on how to use github.dev, etc?

I mentioned it to him in a comment in https://github.com/pi-base/data/pull/686.

**Reply by StevenClontz** [reply]
Yes, absolutely.

I think we're starting to get enough drive-by traffic that it's worth hosting a 'getting started' Zoom every other month. But today we want to explore the new VS Code plugin anyway, so it's a good opportunity for this user to see the tools for making cleaner PRs.
